dkron/scheduler.go

```diff
@@ -72,39 +69,37 @@ func (s *Scheduler) Start(jobs []*Job, agent *Agent) error {
 }
 
 // Stop stops the scheduler effectively not running any job.
```

Comment:
Somewhat agree with the comment, but let's maintain the method name for the sake of consistency with `s.Cron.Stop()`

Reply:
We can reuse the `cron.Stop()` comment
```
// Stop stops the cron scheduler if it is running; otherwise it does nothing.
// A context is returned so the caller can wait for running jobs to complete.
```